Source/JavaScriptCore/jit/JITOperations.cpp

```diff
@@ -1760,12 +1760,14 @@ JSC_DEFINE_JIT_OPERATION(operationNewAsyncGeneratorFunctionWithInvalidatedReallo
 JSC_DEFINE_JIT_OPERATION(operationSetFunctionName, void, (JSGlobalObject* globalObject, JSCell* funcCell, EncodedJSValue encodedName))
 {
     VM& vm = globalObject->vm();
+    auto throwScope = DECLARE_THROW_SCOPE(vm);
```

Comment:
Don't need this because you don't have any exception check (see below) or and you don't throw any exceptions.